specs/paymentrequest.html

```diff
@@ -275,6 +275,15 @@
           list, the payment <code>details</code>, the payment <code>options</code>, and any <a>payment
           method</a> specific <code>data</code>.
         </p>
+
+        <p class="issue" data-number="40" title="How do developers layer additional information in the core payment messages?">
+          How does a developer access the <code>data</code> object?
+          How is <code>data</code> sent to the payment app?
+          Are the core set of payment messages (request/response) extensible?
+          If they are extensible, what is the extensibility mechanism?
+          Is the browser required to understand this extensibility mechanism?
+        </p>
+
```

Comment:
I think this wording is ambiguous and many of these questions are red herrings.

Proposed issue marker text:
```
<p class="issue" data-number="40" title="How does a website pass additional (not payment method specific) data 
to the payment app?">
It is proposed that a conformance criteria for implementations of this API be that any data passed into the 
request is passed on to the payment app unaltered. This would allow extensions of the data schema such 
as the addition of properties that are not documented in this specification or known to implementors such as 
JSON-LD @context or similar to be passed between the website and payment app.
</p>
```